README.md

```diff
@@ -149,13 +149,15 @@ The table below describes all the available configuration values for Waggle Danc
 | `primary-meta-store.metastore-tunnel`                   | No       | See metastore tunnel configuration values below. |
 | `primary-meta-store.latency`                            | No       | Indicates the acceptable slowness of the metastore in **milliseconds** for increasing the default connection timeout. Default latency is `0` and should be changed if the metastore is particularly slow. If you get an error saying that results were omitted because the metastore was slow, consider changing the latency to a higher number.|
 | `primary-meta-store.mapped-databases`                   | No       | List of databases to federate from the primary metastore; all other databases will be ignored. This property supports both full database names and [Java RegEx patterns](https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html) (both being case-insensitive). By default, all databases from the metastore are federated. |
+| `federated-meta-stores[n].database-name-mapping`             | No       | BiDirectional Map of database names and mapped names. key=<database name as known in the primary metastore>, value=<name that should be shown to a client>. See the [Database Name Mapping](#database-name-mapping) section.|
```

Comment:
Looks like this is in here twice, and the info between `< >` isnt showing up in the file.

Reply:
first one should be primary, I'll have a look